.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: test
+
+on:
+    push:
+        branches:
+            - master
+            - release/*
+    pull_request:
+        branches:
+            - master
+            - release/*
+
+jobs:
+    run:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: ['5.6'] #'7.0', '7.1', '7.2', '7.3', '7.4']
+        name: PHP ${{ matrix.php }}
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            # https://github.com/shivammathur/setup-php
+            -   name: Setup
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: xdebug
```

Comment:
Je zou hier `pcov` kunnen gebruiken, ook hier hebben we toch geen xdebug nodig?

Reply:
Als pcov de code coverage ook werkend maakt, dan wel ja.  Kan ik even proberen.

Reply:
Ohja, dat was de reden dat ik op xdebug was gebleven: `PCOV needs PHP >= 7.1.`.  En itertools is nog compatible met php 5.6.

Ik wil heel graag een nieuwe major versie van itertools maken met een heleboel deprecated code minder en wat php 7.2+ reatures.  Maar voor nu is het dus nog xdebug